budget-frontend/tailwind.config.ts

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  content: ['./src/**/*.{js,jsx,ts,tsx}'],
+  theme: {
+    extend: {
+      colors: {
+        primary: '#4CAF50',
+        secondary: '#2196F3',
+        error: '#F44336',
+        background: '#F9FAFB',
+        card: '#FFFFFF',
+        textPrimary: '#333333',
+        textSecondary: '#757575',
+        divider: '#E0E0E0',
+      },
+      fontFamily: {
+        sans: ['Roboto', 'sans-serif'],
+      },
+    },
```

Comment:
for this file, we can also add more things like for example borderRadius,spacing if needed